app/liquid/liquid_renderer.rb

```diff
@@ -116,7 +117,12 @@ def thermometer
   end
 
   def call_tool_data
-    plugin_data.deep_symbolize_keys[:plugins][:call_tool]
+    plugin_data.deep_symbolize_keys[:plugins][:call_tool].map do |data|
+      %i[target_phone target_country_code target_phone_extension target_name target_title].each do |key|
```

Comment:
@osahyoun (and @NealJMD) perhaps a better approach would be to move this to a service, and this function simply replaces the `plugin_data.deep_symbolize_keys[:plugins][:call_tool]` line with a call to `CallCreator`. It also seems that `CallCreator.manual_target` already does pretty much this? (Or I'm not following)

Having this override here makes it difficult to test, whilst having all logic in one service we can ensure that we always get the same object shape, the only difference will be the target list.